components/mpas-ocean/cime_config/config_compsets.xml

```diff
@@ -147,6 +147,16 @@
     <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI%DIB_MPASO%IBDISMFDATMFORCED_DROF%JRA-1p4-2018_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
+    <alias>GMPAS-JRA1p5-DIB-PISMF-DIS</alias>
+    <lname>2000_DATM%JRA-1p5_SLND_MPASSI%DIB_MPASO%IBPISMFDATMFORCED_DROF%JRA-1p5_MALI%DATA_SWAV</lname>
+  </compset>
+
+  <compset>
+    <alias>GMPAS-JRA1p5-DIB-PISMF-SIS</alias>
+    <lname>2000_DATM%JRA-1p5_SLND_MPASSI%DIB_MPASO%IBPISMFDATMFORCED_DROF%JRA-1p5_MALI%SIASTATIC_SWAV</lname>
```

Comment:
@matthewhoffman, are you good with the SIS name here?

Reply:
@xylar , yes, I like the SIS name in the compset, and I agree the GLC component should use the `%SIASTATIC` variant.